src/kaltura-player.js

```diff
@@ -586,6 +595,7 @@ class KalturaPlayer extends FakeEventTarget {
     this._eventManager.listen(this, CoreEventType.ENDED, () => this._onEnded());
     this._eventManager.listen(this, CoreEventType.FIRST_PLAY, () => (this._firstPlay = false));
     this._eventManager.listen(this, CoreEventType.SOURCE_SELECTED, () => (this._sourceSelected = true));
+    this._eventManager.listen(this, CoreEventType.PLAYBACK_ENDED, this._onPlaybackEnded.bind(this));
```

Comment:
I think it'll be better to keep the same syntax. 
arrow function or bind function

Reply:
@yairans please fix

Reply:
already fixed